src/components/Nav.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { AiFillSetting, AiOutlineArrowLeft } from 'react-icons/ai';
+
+const Nav = () => {
+  const { pathname } = useLocation();
+
+  const findPath = () => {
+    let valid = true;
+    let placeholder = '';
+    if (pathname === '/') {
+      valid = false;
+      placeholder = 'Home';
+    }
+    if (valid === true) {
+      return pathname.slice(1, pathname.length);
+    }
+    return placeholder;
+  };
+
+  return (
+    <div className="nav d-flex justify-content-between">
+      <Link data-testid="back-arrow" to="./"><AiOutlineArrowLeft /></Link>
```

Comment:
- [ ] Kindly ensure that the _back_ navigation button is not shown on the _Homepage_. This button performs no function on the _Homepage_.

Reply:
Thanks for the review!